libraries/botbuilder/tests/teamsInfo.test.js

```diff
@@ -26,6 +26,43 @@ class TestContext extends TurnContext {
 }
 
 describe('TeamsInfo', () => {
+    describe('sendMessageToTeamsChannel()', () =>{

```

Comment:
There are negative tests for the args. Can we add a positive test for the appropriate http call actually being made?
You will have to use nock to set up the expected response to the particular http call that is being made.

Reply:
added.